pack/tfa/sha.json

```diff
@@ -0,0 +1,54 @@
+[
+    {
+      "code": "04306",
+      "cost": 1,
+      "deck_limit": 2,
+      "faction_code": "seeker",
+      "illustrator": "Jose Vega",
+      "name": "Vantage Point",
+      "pack_code": "sha",
+      "position": 306,
+      "quantity": 2,
+      "skill_agility": 1,
+      "skill_intellect": 1,
+      "text": "Fast. Play after a location enters play or is revealed during an investigator's turn.\nUntil the end of the active investigator's turn, that location gets -1 shroud. You may move 1 clue to that location from any other location.",
+      "traits": "Insight.",
+      "type_code": "event"
+    },
+    {
+      "code": "04308",
+      "cost": 1,
+      "deck_limit": 1,
+      "exceptional": true,
+      "faction_code": "rogue",
+      "illustrator": "Jose Vega",
+      "name": "Borrowed Time",
+      "pack_code": "sha",
+      "position": 308,
+      "quantity": 2,
+      "skill_agility": 1,
+      "skill_willpower": 1,
+      "slot": "Arcane",
+      "text": "Exceptional.\n[action]: Place 1 resource on Borrowed Time, as a click (to a maximum of 3 clicks on Borrowed Time). Does not provoke attacks of opportunity.\n<b>Forced</b> - When your turn begins: Remove all clicks from Borrowed Time. You may take that many additional actions this turn.",
+      "traits": "Ritual.",
+      "type_code": "asset",
+      "xp": 3
+    },
+    {
+      "code": "04313",
+      "cost": 1,
+      "deck_limit": 2,
+      "faction_code": "survivor",
+      "flavor": "We make our own fate."
```

Comment:
@drawntotheflame looks like a comma is missing on this line

Reply:
thanks mate!